### ISSUE TITLE
running 2.1 but reporting version 2.0 

### ISSUE THREAD

**Issue by n8craig** [ccbogel/QualCoder#288]
**Describe the bug**
I first downloaded Qualcoder 2.0 and ran the file, it reported a newer version. Then I removed 2.0 and downloaded Qualcoder 2.1. It also reports Qualcoder 2.1

**To Reproduce**
I have not tried reproducing on another machine, sorry.

**Expected behavior**
I expect Qualcoder would report running 2.1.

**Screenshots**
Screen shot below shows the notice. Also to the right, one can just see the file explorer showing the current directory is `QualCoder-2.1`.

![2020-11-08 18_47_53-Window](https://user-images.githubusercontent.com/62002677/98491943-9e356100-21f3-11eb-926e-fb96b37d1cae.jpg)

**Desktop (please complete the following information):**
- Windows 10
- Python 3.7.7 under Anaconda

**Additional context**
Not sure if I've done something wrong or this is an issue under Win. It's a minor matter, but wanted to report back and see if there was something I'm missing.

I just want to say, this is a _very cool_ piece of software. I've been looking for something like this for a while. Really excited to dig in and start working with it. Thank you so much.

**Comment by ccbogel** [comment]
OK. That is odd. I just now downloaded the zip folder from the releases: 
https://github.com/ccbogel/QualCoder/releases/tag/2.1
I ran it and it does not show 2.0. It correctly shows 2.1
I also checked the line in the qualcoder.py file - on line 69 which is definitely QualCoder 2.1

Yes you are correct in that this is a minor issue.

Im glad you think its 'very cool'  :)
I think it works pretty well, but occasionally issues crop up. One issue that I will rectify in next week or 2 in another release is the entry and typing in to text files created within QualCoder. - Basically does not work, due to a minor coding issue, which an update will fix. However, the importation of text files and using them is fine.

It would be great to continue to get feedback so that I can gradually improve it. It is a hobby project, so I dont always have the time to work on it. Also limited resources - so I cant test on many platforms (Ubuntu first and Windows secondly). It would also be really good to get mini-projects exported in the REFI-QDA format so that I can do what  I can to improve their importation.
